package.json

```diff
@@ -24,20 +24,20 @@
     "express": "^4.18.2",
```

Comment:
We're not using `express` [anymore](https://github.com/wesleytodd/express-openapi/pull/59/files#diff-5bb8db779819ddef5956a5d9d5949c05ef7445237656ca37bf2f02720271440bL6). So, I think I should remove this package. @wesleytodd could you please confirm.

Reply:
Nevermind, I misread the code. If `EXPRESS_MAJOR` is not defined `express` will be used. Disregard my previous comment.